pkg/actors/actors.go

```diff
@@ -461,7 +464,11 @@ func (a *actorsRuntime) callLocalActor(ctx context.Context, req *invokev1.Invoke
 			reentrancyID = &headerValue.GetValues()[0]
 		} else {
 			reentrancyHeader := fasthttp.RequestHeader{}
-			uuid := uuid.New().String()
```

Comment:
Note that methods like `uuid.New()` can panic in certain cases (e.g. if the kernel doesn't have enough entropy). Hence why the change.